Includes/menu.cpp

```diff
@@ -137,27 +137,38 @@ void MenuNode::moveSelection(int delta, bool allowWrap) {
   }
 }
 
+/******************************************************************************************
+                               MenuNonInteractive
+******************************************************************************************/
+MenuNonInteractive::MenuNonInteractive(MenuNode *parent, std::string const& label) : MenuNode(parent, label) {}
+
+void MenuNonInteractive::execute(Menu *) {
+  // Do nothing, this node cannot be entered.
+}
+
 /******************************************************************************************
                                    MenuXbe
 ******************************************************************************************/
 MenuXbe::MenuXbe(MenuNode *parent, std::string const& label, std::string const& paths) :
   MenuNode(parent, label) {
+
   size_t path_start = 0;
   for (size_t path_end = paths.find(PATH_DELIMITER, path_start);
-       path_end != std::string::npos;
-       path_end = paths.find(PATH_DELIMITER, path_start)) {
-    
-      std::string subpath = paths.substr(path_start, path_end - path_start);
-      findXBE(subpath, this);
-      path_start = path_end + 1;
-  }
-  std::string path = paths.substr(path_start);
+    path_end != std::string::npos;
+    path_end = paths.find(PATH_DELIMITER, path_start)) {
```

Comment:
Indentation change

Reply:
Hopefully fixed. I'm not sure if the preferred style for this project is to align under the first clause of the `for` loop or to indent 4 spaces (I went with the former). Please let me know if I got it wrong!

Reply:
2 space indentation and/or align under first clause. :+1: